docs/pages/docs/guides/images-and-files.md

```diff
@@ -3,11 +3,11 @@ title: "Images and Files"
 description: "Learn how to add Images and Files to your project using Keystone’s Storage configuration"
 ---
 
-Keystone [fields](../fields/overview) include the `image` and `file` types. You can use them to reference and (if required) serve images and/or files from Keystone. This guide will show you how to configure images and files in your Keystone system so you can store assets either locally or [Amazon S3 storage](https://aws.amazon.com/s3/).
+Keystone [fields](../fields/overview) include the `image` and `file` types. You can use them to reference and (if required) serve images and/or files from Keystone. This guide will show you how to configure images and files in your Keystone system so you can store assets either locally or [Amazon S3 storage](https://aws.amazon.com/s3/) or DigitalOcean Spaces (https://www.digitalocean.com/products/spaces).
 
 ## How asset storage works in Keystone
 
-Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` (by [Amazon](https://aws.amazon.com/s3/)) depending on your use case.
+Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` (by [Amazon](https://aws.amazon.com/s3/)) and `S3-Compatible Cloud Object Storage` (by [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces)) depending on your use case.
```

Comment:
```suggestion
Keystone manages file and image assets through a `storage` object you define in Keystone’s [configuration file](/docs/config/config). Any number of stores can be set up within the `storage` object, and you can mix and match between `local` and `s3` compatible providers (akin to [Amazon S3](https://aws.amazon.com/s3/, or [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces)).
```

We have no preference for a provider, only that it is compatible.  Maybe this is an improvement?

Reply:
I'm weary of adding many S3 providers to this section, but don't want to bias either.  Maybe we can use generic language after _one_ section that lists a number of _different_ providers.

Reply:
Yes, Thank You!